clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/deploy/description/AbstractAppEngineCredentialsDescription.groovy

```diff
@@ -20,5 +20,6 @@ import com.netflix.spinnaker.clouddriver.appengine.security.AppEngineNamedAccoun
 import com.netflix.spinnaker.clouddriver.security.resources.CredentialsNameable
 
 abstract class AbstractAppEngineCredentialsDescription implements CredentialsNameable {
```

Comment:
Only as a marker that I don't expect it to be instantiated, there isn't a corresponding converter, validator, or operation, etc.